spec/models/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe '#update_comment_counter' do
+    it 'updates the comments_counter of the associated post' do
+      user = create(:user) # Assuming you have a factory for User
+      post = create(:post, user:) # Assuming you have a factory for Post
+      comment = create(:comment, user:, post:) # Assuming you have a factory for Comment
+
+      expect { comment.update_comment_counter }.to change { post.reload.comments_counter }.by(1)
+    end
+  end
+end
```

Comment:
- [ ] As previous reviewer mentioned [here](https://github.com/Nessrine88/Blog-App/pull/3#discussion_r1382311166), kindly make sure that all of your tests are passing. Currently I'm getting the following errors 👇 Please make sure to run your tests before asking for another review

_Testing individual files_

`comment_spec.rb`
![image](https://github.com/Nessrine88/Blog-App/assets/106119105/a57829c5-4132-4e2e-b47f-8f70fbe26753)

`like_spec.rb`
![image](https://github.com/Nessrine88/Blog-App/assets/106119105/c8d86a80-9ced-4b0f-b03e-4d69bb2fe47e)

`post_spec.rb`
![image](https://github.com/Nessrine88/Blog-App/assets/106119105/532e5b48-7323-4286-b8e7-5f0359821859)

`user_spec.rb`
![image](https://github.com/Nessrine88/Blog-App/assets/106119105/9406d057-fd13-4ab2-a51a-dfbd99e2532a)

_Testing all files_
![image](https://github.com/Nessrine88/Blog-App/assets/106119105/7ea92b66-9a5c-4414-9b90-145cfd081d05)